ir-engine/ir-engine
# IR-5010 component dropdown persist

## Summary
component dropdown persist

## References
[https://tsu.atlassian.net/browse/IR-5010](https://tsu.atlassian.net/browse/IR-5010)

## QA Steps
inspect an entity in the properties panel
observe the component properties start out expanded the first time you look at an entity in the properties panel
minimize a component(s)
change your selection to a different entity
go back to the original
observe the component(s) are in the expanded or collapsed state you left them in before
